spec/services/waste_carriers_engine/determine_area_service_spec.rb

```diff
@@ -37,8 +37,8 @@ module WasteCarriersEngine
         context "with no match found" do
           let(:response) { instance_double(DefraRuby::Area::Response, successful?: false, error: DefraRuby::Area::NoMatchError.new) }
 
-          it "returns 'Not found'" do
-            expect(described_class.run(coordinates)).to eq "Not found"
+          it "returns 'Outside England'" do
+            expect(described_class.run(coordinates)).to eq "Outside England"
```

Comment:
Not sure if that's correct to say that address is Outside England if NoMAtchError is returned. I believe same error is returned when address lookup just fails